descriptors/ModuleDescriptor-template.json

```diff
@@ -15,7 +15,7 @@
     },
     {
       "id": "loan-storage",
-      "version": "6.2",
+      "version": "6.3",
```

Comment:
Please, update version in the RAML file as well.

Reply:
it's 6.3 already there.

Reply:
Was that a typo in a previous change or has a version been missed in the module descriptor?

Reply:
I think that previously, when the version was increased to 6.3 in raml, was not increased to 6.3 in module descriptor. @marc-johnson Do you think that it's a good idea to bump the version up to 6.4 in both files?

Reply:
Yes, please, if there was meant to be a 6.3 and it was missed, please move both onto 6.4

(That likely means that nothing binds directly to 6.3 otherwise there would've been environmental build failures)

Reply:
Done.